src/content/whats-new/2021/04/distributed-tracing-mobile.md

```diff
@@ -0,0 +1,20 @@
+---
+title: 'Distributed tracing for Mobile'
+summary: 'Get visibility to the entire journey of requests, originating in your mobile app as they travel through distributed systems'
+releaseDate: '2021-04-14'
+learnMoreLink: 'https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-mobile-and-dt/'
+---
+
+New Relic mobile monitoring agents now support distributed tracing! This new feature enables you to see your mobile app activity and connects it directly to the related services that power your app’s experience. By having this enabled, you are now able to see your mobile front-end spans and back-end spans in the same waterfall view.
```

Comment:
Since this doesn't apply to the old agents, how about inserting something like this: "Our new mobile monitoring agents now support distributed tracing!"